python/doc/examples/meta_modeling/kriging_metamodel/plot_gpr_advanced.py

```diff
@@ -1,6 +1,6 @@
 """
-Advanced Kriging
-================
+Advanced Gaussian process regression
+====================================
 """
 
```

Comment:
some examples such as this one are referenced in python/test/CMakeLists.txt, so it has to be modified too:
`set_tests_properties (pyinstallcheck_example_plot_kriging_advanced PROPERTIES DISABLED TRUE)`